src/lib/keypom.ts

```diff
@@ -190,7 +207,21 @@ class KeypomJS {
     return null;
   };
 
-  getDrops = async ({ accountId, start, limit }) => await getDrops({ accountId, start, limit });
+  getDrops = async ({ accountId, start, limit }) => {
+    /** Get Drops caching logic */
+    if (
+      Date.now() > this.dropStore.getDropsExpiryTime ||
+      start !== this.dropStore.getDropsLastPage
+    ) {
+      const newGetDropsExpiryTime = new Date(Date.now() + CACHE_MAX_AGE).getTime();
+      this.dropStore.getDropsExpiryTime = newGetDropsExpiryTime;
+
+      this.dropStore.getDropsLastPage = start;
```

Comment:
```suggestion
      this.dropStore.getDropsLastPageIndex = start;
```

Reply:
`getDropsLastPage` a bit confused by this, is it referencing the current last item index?